src/test/java/com/epam/jwd/action/result/TestResult.java

```diff
@@ -1,4 +1,4 @@
-package com.epam.jwd.result;
+package com.epam.jwd.action.result;
 
 public class TestResult {
```

Comment:
maybe it will be clearer if the methods "isBaseOnPlane() " and "isCube" were renamed in "getIsBaseOnPlane()" and "getIsCube" accordingly